templates/python/.github/workflows/pre-commit.yaml

```
@@ -0,0 +1,52 @@
+name: Pre-commit
+
+on:
+  push:
+    # branches: [master]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
+  pull_request:
+    branches: [master]
+
+jobs:
+  pre-commit:
+    name: pre-commit
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8"]
+    defaults:
+      run:
+        working-directory: ${{ env.working-directory }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # - name: Install packages
+      #   run: sudo apt-get -y install python3-dev graphviz libgraphviz-dev pkg-config python3-wheel
+
+      - name: Set up Poetry
+        uses: bakdata/ci-templates/actions/python-setup-poetry@v1.5.1
```

latest version of this action

```suggestion
        uses: bakdata/ci-templates/actions/python-setup-poetry@v1.5.3
```

---

done